recipes/conversation_view/feed_pill/feed_item_pill.stories.js

```diff
@@ -62,42 +64,28 @@ const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
   DtRecipeFeedItemPillDefaultTemplate,
 );
 
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtRecipeFeedItemPillVariantsTemplate,
+);
+
 // Stories
 export const Default = {
   render: DefaultTemplate,
-  parameters: { options: { showPanel: false }, a11y: { disable: true }, controls: { disable: true } },
-};
-
-export const DefaultToggledVariant = {
-  render: DefaultTemplate,
-  parameters: Default.parameters,
-  args: {
-    defaultToggled: true,
-  },
-};
-
-export const DefaultToggledVariantNonToggleable = {
-  render: DefaultTemplate,
-  parameters: Default.parameters,
-  args: {
-    toggleable: false,
-    defaultToggled: true,
-  },
-};
-
-export const AiBorderGradient = {
-  render: DefaultTemplate,
-  parameters: Default.parameters,
-  args: {
-    borderColor: 'ai',
+  parameters: {
+    options: { showPanel: false },
+    a11y: { disable: true },
+    percy: {
+      args: { seed: 'seed' },
+    },
```

Comment:
- The default story should always show the panel, just hide it on the variants story.
- The percy argument `seed` is just for when we're using avatars, so they don't randomly change the color on every visual test, given that, you don't need it on this story.

```suggestion
  parameters: {
    a11y: { disable: true }
  }
```

Reply:
Oh makes sense, thought it was added to allow percy to run the visual test for the component, I was not able to access percy so didn't know that. Thanks.